.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
Maybe add `trim_trailing_whitespace = true`? Even though may editor do this my default, I still good to have this explicit.

Reply:
I didn't know that rule existed. I'll add that and make a note to go add that to all of the other repositories as well